core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java

```diff
@@ -137,17 +138,23 @@
   abstract class WriteRequestInternal implements Closeable {
     /** This ID can either be block ID or temp UFS file ID. */
     final long mId;
+    /** The session id associated with all temporary resources of this request. */
     final long mSessionId;
 
-    WriteRequestInternal(long id, long sessionId) {
+    WriteRequestInternal(long id) {
       mId = id;
-      mSessionId = sessionId;
+      mSessionId = IdUtils.getRandomNonNegativeLong();
```

Comment:
nit: I would suggest to have a wrapper method called `IdUtils.createSessionId` which calls `IdUtils.getRandomNonNegativeLong()` internally. Same comment apply to other references of `IdUtils.getRandomNonNegativeLong();` in this PR